### DISCUSSION TITLE
Updating the list

### DISCUSSION THREAD

**Discussion by davisonio** [davisonio/awesome-gif#12] [General]
Many of the resources in this list are outdated, and better more awesome projects may exist for them. If you know of any better resources we could add please suggest.
Also the scripts are old and some of them may not work anymore, we should test these!

**Comment by rarelygoeshere** [comment]
Well, it looks like this repo seems to have been left abandoned. I might not know of any new resources for this repo, but nonetheless, I have recently examined all the links awesome gifs got, and pasted the outdated and dead ones on this txt file. I hope this will be of use to you. I didn't include any links where the last update was in 2020-2023 cuz I didn't felt it was too out of date.
[awesome gif dead and outdated resources.txt](https://github.com/davisonio/awesome-gif/files/12263229/awesome.gif.dead.and.outdated.resources.txt)

**Reply by davisonio** [reply]
Nice I'll check it out
